### PR TITLE
UINV-6 Add an address list to settings -> Tenant -> Addresses

### PR DESCRIPTION
**Pull request by NikitaSedyx**
https://issues.folio.org/browse/UINV-6

Purpose: The system needs a global list of addresses that relate directly to the institution. The invoice will draw on that list for Bill to and Ship to

### CODE REVIEW

File: src/settings/index.js [hunk 1]
```diff
@@ -25,6 +26,12 @@ class Organization extends React.Component {
       {
         label: <FormattedMessage id="ui-tenant-settings.settings.general.label" />,
         pages: [
+          {
+            route: 'addresses',
+            label: <FormattedMessage id="ui-tenant-settings.settings.addresses.label" />,
+            component: Addresses,
+            perm: 'ui-tenant-settings.settings.addresses',
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/settings/index.js]: oh, I forgot about this, thank you!

**Comment by NikitaSedyx** [comment]
@zburke as for #1 you need to have code from `smart-components` is not merged

as for #2 it's good question for PO, but as for me it can be a separate story in this case to not block requests

File: src/settings/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/settings/index.js]: updated

**Comment by zburke** [comment]
> @zburke as for #1 you need to have code from smart-components is not merged

In this case, you need to update the `@folio/stripes` dependency version in `package.json` as well.

**Comment by NikitaSedyx** [comment]
@zburke updated depths

**Comment by NikitaSedyx** [comment]
@zburke I think you understand that 2.7 is not available and build for this repo will be broken in this case

plus we use `^`, so we should be ok with it

plus updated `stripes` in `peerDependencies`, I don't think other depts should be updated in this request

**Comment by zburke** [comment]
>  I think you understand that 2.7 is not available and build for this repo will be broken in this case

Yes, it is my expectation that the PR will be unmergeable until v2.7 is released because the CI job that attempts to install the module will fail during the dependency installation. This PR is blocked on new releases of stripes-smart-components and stripes. 

> plus we use ^, so we should be ok with it

The caret sets an upper bound on the compatible version. You still need to provide a legitimate lower bound, which in this case will be v2.7.0 because this PR relies on features not available in `@folio/stripes/components` v2.6.0.

**Comment by NikitaSedyx** [comment]
@zburke in this case please review https://github.com/folio-org/stripes-smart-components/pull/516

and when can we make `2.7` release?

**Pull request merged**